.github/workflows/early_access.yml

```
@@ -12,13 +12,13 @@ jobs:
     if: startsWith(github.event.head_commit.message, '[maven-release-plugin]') != true
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
 
     - name: Set up JDK 11
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3.13.0
       with:
         java-version: 11
-        distribution: 'adopt'
+        distribution: 'temurin'
```

You meant Microsoft, right? 😁 

```suggestion
        distribution: 'microsoft'
```